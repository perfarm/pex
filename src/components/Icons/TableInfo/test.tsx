import React from 'react';

import { render } from '@testing-library/react';

import { TableInfo } from '.';

describe('TableInfo', () => {
  test('should match snapshot', () => {
    const { container } = render(<TableInfo />);

    expect(container).toMatchSnapshot();
  });
});
