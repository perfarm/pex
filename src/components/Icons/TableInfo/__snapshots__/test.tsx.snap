// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableInfo should match snapshot 1`] = `
<div>
  <svg
    class="PJLV"
    fill="none"
    height="18"
    viewBox="0 0 80 80"
    width="18"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      class="PJLV"
      clip-rule="evenodd"
      d="M9 18.0312C9 17.617 9.33579 17.2812 9.75 17.2812H69C69.4142 17.2812 69.75 17.617 69.75 18.0312V47.0055C69.6255 47.0018 69.5004 47 69.375 47C67.2954 47 65.3344 47.5078 63.6093 48.4062H27.7813V61.7188H57.0713C57.1629 62.2302 57.2857 62.7309 57.4375 63.2188H12.2188C11.3651 63.2188 10.5464 62.8796 9.94275 62.276C9.33912 61.6724 9 60.8537 9 60V18.0312ZM10.5 33.5938V46.9062H26.2813V33.5938H10.5ZM10.5 32.0938V18.7812H68.25V32.0938H10.5ZM27.7813 33.5938V46.9062H68.25V33.5938H27.7813ZM26.2813 61.7188V48.4062H10.5V60C10.5 60.4558 10.6811 60.893 11.0034 61.2153C11.3257 61.5377 11.7629 61.7188 12.2188 61.7188H26.2813ZM69.3535 51.2266C64.7972 51.2266 61.1035 54.9202 61.1035 59.4766C61.1035 64.0329 64.7972 67.7266 69.3535 67.7266C73.9099 67.7266 77.6035 64.0329 77.6035 59.4766C77.6035 54.9202 73.9099 51.2266 69.3535 51.2266ZM59.6035 59.4766C59.6035 54.0918 63.9687 49.7266 69.3535 49.7266C74.7383 49.7266 79.1035 54.0918 79.1035 59.4766C79.1035 64.8613 74.7383 69.2266 69.3535 69.2266C63.9687 69.2266 59.6035 64.8613 59.6035 59.4766ZM69.3535 54.2266C69.7677 54.2266 70.1035 54.5623 70.1035 54.9766V60.2266C70.1035 60.6408 69.7677 60.9766 69.3535 60.9766C68.9393 60.9766 68.6035 60.6408 68.6035 60.2266V54.9766C68.6035 54.5623 68.9393 54.2266 69.3535 54.2266ZM69.3533 64.7265C69.9747 64.7265 70.4783 64.2228 70.4783 63.6015C70.4783 62.9802 69.9747 62.4765 69.3533 62.4765C68.732 62.4765 68.2283 62.9802 68.2283 63.6015C68.2283 64.2228 68.732 64.7265 69.3533 64.7265Z"
      fill="#898989"
      fill-rule="evenodd"
    />
  </svg>
</div>
`;
